tests/string/string.php

```diff
@@ -33,12 +33,13 @@
 $string = "foo\u{ff}bar";
 $string = "foo\ubar";
 $string = "foo\u";
-$string = "He drank some $juice juice.";
-$string = "He drank some {$juice} juice.";
-$string = "He drank some ${juice} juice.";
-$string = "He drank some ${$juice} juice.";
-$string = "He drank some juice made of ${juice}s.";
-$string = "He drank some $juices[0] juice.";
+$string = "simple $encapsed";
+$string = "simple $encapsed[0]";
+$string = "encapsed variable ${name}.";
+$string = "encapsed variable {$value}.";
+$string = "encapsed offset {$value[0]}.";
+$string = "encapsed {$method->call()}.";
+$string = "combined ${$encapsed}.";
```

Comment:
@evilebottnawi i had to change some of the texts for debugging - got some errors when running with `AST_COMPARE=1` that were hard to track down when all the strings were so similar 😉